src/main/kotlin/app/revanced/patches/reddit/testing/patch/TotalAdPostPatch.kt

```diff
@@ -0,0 +1,179 @@
+package app.revanced.patches.reddit.testing.patch
```

Comment:
Please move the patch to the correct package, take other patches as an example.

Reply:
Forgor to fix :skull: